### DISCUSSION TITLE
3d Cab Not showing up

### DISCUSSION THREAD

**Discussion by DEVDD960** [perpetualKid/FreeTrainSimulator#122] [General]
Many stocks have 3d cabs these are not shown up in the latest version. Any idea how to make them work properly???

**Comment by perpetualKid** [comment]
which version/update channel are you using? 
There were some changes/fixes related to 3D cabs just few days ago, which are in latest development builds but not yet elevated into RC/Release builds.

**Reply by DEVDD960** [reply]
yeah..i am on the public release channel...should i switch to public-pre release or Developer-test release??? Channel?? 

And I am currently on latest public Release version: 1.4.12.1

**Reply by perpetualKid** [reply]
yes, if you switch to developer test channel, which is currently on v1.4.13-dev.43, 3D Cabs should work again

**Reply by DEVDD960** [reply]
Yeah..its ..working...I really wish to play on this version...but most of the Acts are crashing on it...Also reshade is not working on this version, I hope all these issues get fixed soon on it... I tried on the developer release...3d cab worked perfectly, but when I started an Act, it got Crashed after some time...no idea why..

**Reply by perpetualKid** [reply]
I haven't seen or heard about specific issues, so it would be great to learn what is not working. ideally attach a logfile, and/or point to the route/activity if they are publicly available for download.
Also, Reshade should work but may need some tweaks to ensure bitness as there where issues in the past with 64bit OR.

**Reply by DEVDD960** [reply]
Routes/Acts..are paid.. I will attach the log file next time when I try it...For reshade its not loading, I tried installing it , but still no luck, the reshade doesnt comes up during the loading of acts or Exploring the route, as it usually comes in other mg versions or official versions, same issue is that with the latest official build of OR (1.5.1) ..reshade doesnt loads up
